### PR TITLE
Update 2.2.0 changelog for b2

### PR DESCRIPTION
**Pull request by jedcunningham**

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -119,6 +133,33 @@ Improvements
...
+- Properly handle ti state difference between executor and scheduler (#17819)
+- Fix stuck "queued" tasks in KubernetesExecutor (#18152)
+- Don't permanently add zip DAGs to ``sys.path`` (#18384)
+- Fix random deadlocks in MSSQL database (#18362)
+- Deactivating DAGs which have been removed from files (#17121)
+- Dag bulk_sync_to_db dag_tag only remove not exists (#8231)
+- Graceful scheduler shutdown on error (#18092)
+- Fix mini scheduler not respecting ``wait_for_downstream`` dep (#18338)
+- Pass exception to ``run_finished_callback`` for Debug Executor (#17983)
+- Make `XCom.get_one` return full, not abbreviated values (#18274)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Make ``XCom.get_one`` return full, not abbreviated values (#18274)
```

File: CHANGELOG.txt [hunk 2]
```diff
@@ -119,6 +133,33 @@ Improvements
...
+- Fix web view rendering errors without a DAG run (#18244)
+- Use try/except when closing temporary file in task_runner (#18269)
+- show next run if not none (#18273)
+- Fix dag_run FK check in pre-db upgrade (#18266)
+- Fix DB session handling in XCom.set. (#18240)
+- Sort adopted tasks in _check_for_stalled_adopted_tasks method (#18208)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Sort adopted tasks in ``_check_for_stalled_adopted_tasks`` method (#18208)
```

File: CHANGELOG.txt [hunk 3]
```diff
@@ -119,6 +133,33 @@ Improvements
...
+- Dag bulk_sync_to_db dag_tag only remove not exists (#8231)
+- Graceful scheduler shutdown on error (#18092)
+- Fix mini scheduler not respecting ``wait_for_downstream`` dep (#18338)
+- Pass exception to ``run_finished_callback`` for Debug Executor (#17983)
+- Make `XCom.get_one` return full, not abbreviated values (#18274)
+- Fix web view rendering errors without a DAG run (#18244)
+- Use try/except when closing temporary file in task_runner (#18269)
+- show next run if not none (#18273)
+- Fix dag_run FK check in pre-db upgrade (#18266)
+- Fix DB session handling in XCom.set. (#18240)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Fix DB session handling in ``XCom.set`` (#18240)
```

File: CHANGELOG.txt [hunk 4]
```diff
@@ -119,6 +133,33 @@ Improvements
...
+- Fix deleting of zipped Dags in Serialized Dag Table (#18243)
+- Return explicit error on user-add for duplicated email (#18224)
+- Remove loading dots even when last run data is empty (#18230)
+- BugFix: Wipe ``next_kwargs`` and ``next_method`` on task failure (#18210)
+- Add heartbeat to TriggererJob (#18129)
+- Swap dag import error dropdown icons (#18207)
+- Automatically create section when migrating config (#16814)
+- sets encoding to utf-8 by default while reading task logs (#17965)
+- Set task state to failed when pod is DELETED while running (#18095)
+- Apply parent dag permissions to subdags. (#18160)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Apply parent dag permissions to subdags (#18160)
```

File: CHANGELOG.txt [hunk 5]
```diff
@@ -119,6 +133,33 @@ Improvements
...
+- show next run if not none (#18273)
+- Fix dag_run FK check in pre-db upgrade (#18266)
+- Fix DB session handling in XCom.set. (#18240)
+- Sort adopted tasks in _check_for_stalled_adopted_tasks method (#18208)
+- Fix external_executor_id not being set for manually run jobs (#17207)
+- Fix deleting of zipped Dags in Serialized Dag Table (#18243)
+- Return explicit error on user-add for duplicated email (#18224)
+- Remove loading dots even when last run data is empty (#18230)
+- BugFix: Wipe ``next_kwargs`` and ``next_method`` on task failure (#18210)
+- Add heartbeat to TriggererJob (#18129)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Add heartbeat to ``TriggererJob`` (#18129)
```

File: CHANGELOG.txt [hunk 6]
```diff
@@ -119,6 +133,33 @@ Improvements
 
 Bug Fixes
 """""""""
+- Properly handle ti state difference between executor and scheduler (#17819)
+- Fix stuck "queued" tasks in KubernetesExecutor (#18152)
+- Don't permanently add zip DAGs to ``sys.path`` (#18384)
+- Fix random deadlocks in MSSQL database (#18362)
+- Deactivating DAGs which have been removed from files (#17121)
+- Dag bulk_sync_to_db dag_tag only remove not exists (#8231)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Dag ``bulk_sync_to_db`` ``dag_tag`` only remove not exists (#8231)
```

File: CHANGELOG.txt [hunk 7]
```diff
@@ -37,6 +44,13 @@ New Features
 
 Improvements
 """"""""""""
+- Require can_edit on DAG privileges to modify TaskInstances and DagRuns (#16634)
+- Make Kubernetes job description fit on one log line (#18377)
+- Always draw borders if task instance state is null or undefined (#18033)
+- Inclusive Language (#18349)
+- Improved log handling for zombie tasks (#18277)
+- Adding Variable.update method and improving detection of variable key collisions (#18159)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Adding ``Variable.update`` method and improving detection of variable key collisions (#18159)
```

File: CHANGELOG.txt [hunk 8]
```diff
@@ -1,13 +1,20 @@
...
 New Features
 """"""""""""
 
 - AIP-39: Handle DAG scheduling with timetables (#15397)
 - AIP-39: ``DagRun.data_interval_start|end`` (#16352)
+- Add docs for AIP 39: Timetables (#17552)
 - AIP-40: Add Deferrable "Async" Tasks (#15389)
+- Add a Docker Taskflow decorator (#15330)
 - Add Airflow Standalone command (#15826)
+- Production-level support for MSSQL (#18382)
```
Comment (kaxil) [inline CHANGELOG.txt]: Probably worth moving to Misc/Internal

```suggestion
- Dockerfile: Production-level support for MSSQL (#18382)
```

Comment (kaxil) [reply CHANGELOG.txt]: wdyt? as it is not really Airflow's new feature

File: CHANGELOG.txt [hunk 9]
```diff
@@ -173,6 +214,14 @@ Bug Fixes
 
 Doc only changes
 """"""""""""""""
+- Explain scheduler fine-tuning better (#18356)
+- Added example JSON for airflow pools import (#18376)
+- add sla_miss_callback section to the documentation (#18305)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Add ``sla_miss_callback`` section to the documentation (#18305)
```

File: CHANGELOG.txt [hunk 10]
```diff
@@ -230,6 +279,13 @@ Doc only changes
 
 Misc/Internal
 """""""""""""
+- Update boto3 to <1.19 (#18389)
+- Improve coverage for airflow.security.kerberos module (#18258)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Improve coverage for ``airflow.security.kerberos module`` (#18258)
```

File: CHANGELOG.txt [hunk 11]
```diff
@@ -230,6 +279,13 @@ Doc only changes
 
 Misc/Internal
 """""""""""""
+- Update boto3 to <1.19 (#18389)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
- Update ``boto3`` to ``<1.19`` (#18389)
```

File: CHANGELOG.txt [hunk 12]
```diff
@@ -230,6 +279,13 @@ Doc only changes
...
+- Improve coverage for airflow.security.kerberos module (#18258)
+- Fix kinesis test (#18337)
+- Fix provider test acessing importlib-resources (#18228)
+- Silence warnings in tests from using SubDagOperator (#18275)
+- Remove Brent from Collaborators (#18182)
```
Comment (kaxil) [inline CHANGELOG.txt]: We can remove this, not worth Changelog.

```suggestion
```

File: CHANGELOG.txt (hunk 8 above)
Comment (jedcunningham) [reply CHANGELOG.txt]: Probably should be a docs-only change instead on second look.

**Review by kaxil — APPROVED** [review]

**Comment by jedcunningham** [comment]
Docs build failure is unrelated and will be fixed by #18419.

**Pull request merged**